### ISSUE TITLE
Add Support for NavController

### ISSUE THREAD

**Issue by hgh-bibliophile** [Droppers/AnimatedBottomBar#16]
Hello, I really like the design of your library, and I was wondering if you could add the ability to easily set it up with a NavController, as the library I was previously using, [ibrahimsn98/SmoothBottomBar](https://github.com/ibrahimsn98/SmoothBottomBar), has this functionality. I found how they implemented it, and was able to modify their code to work with your library. I have included the file I modified instead of trying to create a fork of your library, as I am not familiar with how your library works internally. 
I don't know if this helps, but I belive this is how they used the NavigationComponentHelper in their main class:
```kotlin
    fun setupWithNavController(menu: Menu, navController: NavController){
        NavigationComponentHelper.setupWithNavController(menu,this,navController)
    }
```
Thank you for considering this!
[NavigationComponentHelper.txt](https://github.com/Droppers/AnimatedBottomBar/files/4944654/NavigationComponentHelper.txt)

**Comment by Droppers** [comment]
Will add this as soon as possible, even though it's a little late it's still a useful feature.

**Comment by Droppers** [comment]
This is now available, released version 1.0.9: `implementation 'nl.joery.animatedbottombar:library:1.0.9'`.
